### DISCUSSION TITLE
Creating Row Support

### DISCUSSION THREAD

**Discussion by r20** [KevinVandy/mantine-react-table#88] [General]
I see evidence that creating rows is either newly supported or it's being added.
E.g. Feature Guide has "Editing and Creating" and there are table options like createDisplayMode and onCreatingRowSave.  
However, there are no examples and looking in code it seems it's not all there yet.
Is that right?
Is there an ETA on support for this?

Thanks.

**Comment by KevinVandy** [comment]
There are 4 detailed examples here https://www.mantine-react-table.com/docs/examples/editing-crud

<img width="819" alt="image" src="https://github.com/KevinVandy/mantine-react-table/assets/28243511/0eb9c70b-dda8-4345-85bd-25bac01c16bd">

These have all been added in the last 2 weeks. I'm currently using these features very heavily at my startup now. Getting the full docs up soon, but these examples are pretty much going to be what the docs are.

**Reply by r20** [reply]
Thanks!
